### PR TITLE
Cid faq cache

### PR DESCRIPTION
**Pull request by awssteph**
added an FAQ on cache error

### CODE REVIEW

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 1]
```diff
@@ -279,6 +279,31 @@ This error is caused by there being too many [data source connectors](https://do
...
 
+{{% /expand%}}
+
+## How do I fix the ‘product_cache_engine’ cannot be resolved error? 
+
+When attempting to deploy the dashboard, some users get an error that states ‘product_cache_engine’ cannot be resolved. 
+{{%expand "Click here to expand answer" %}}
+
+This view is dependent on having or historically having an RDS database instance and an ElastiCache cache instance run in your organization. If you get the error that the column ‘product_database_engine’ or product_deployment_option does not exist, then you do not have any RDS database instances running. There are two options to resolve this.
```
Comment (awsdunc) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:289]: Should `product_database_engine` and `product_deployment_option` both be in the same backticks?

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 2]
```diff
@@ -279,6 +279,31 @@ This error is caused by there being too many [data source connectors](https://do
...
+
+### Option 1
+ To make this column show up in the CUR spin up a database in the RDS service, let it run for a couple of minutes and in the next integration of the crawler the column will appear. If you get the error that the column ‘product_cache_engine’ does not exist, then you do not have any ElastiCach cache instances running. To make this column show up in the CUR spin up an ElastiCache cache instance in the ElastiCache service, let it run for a couple of minutes and in the next integration of the crawler the column will appear. You can verify this by running the Athena query: SHOW COLUMNS FROM tablename - and replace the tablename accordingly after selecting the correct CUR database in the dropdown on the left side in the Athena view.
```
Comment (awsdunc) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:292]: ElastiCach missing e

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 3]
```diff
@@ -279,6 +279,31 @@ This error is caused by there being too many [data source connectors](https://do
...
+
+### Option 2
+Follow the below steps to remove the colum. Be aware this will mean if you do add ElastiCache instances to your accounts you should put this back.
+
+1. In Amazon Athena click 'Show Edit Query' for kpi_instance_all
+2. Remove *‘product_cache_engine’* and remove the last *Group by* number
+3. Run query
+4. IF you are running from CLoudShell re-run the deploy command
+5. Go to AWS Quicksight
```
Comment (awsdunc) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:301]: Amazon Quicksight

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 4]
```diff
@@ -279,6 +279,31 @@ This error is caused by there being too many [data source connectors](https://do
...
+
+### Option 2
+Follow the below steps to remove the colum. Be aware this will mean if you do add ElastiCache instances to your accounts you should put this back.
+
+1. In Amazon Athena click 'Show Edit Query' for kpi_instance_all
+2. Remove *‘product_cache_engine’* and remove the last *Group by* number
+3. Run query
+4. IF you are running from CLoudShell re-run the deploy command
+5. Go to AWS Quicksight 
+6. Find the kpi_instance_all dataset
```
Comment (awsdunc) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:302]: kpi_instance_all in backticks?

**Review by awsdunc — CHANGES_REQUESTED** [review]

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md (hunk 2 above)
Comment (awssteph) [reply content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md]: Done

**Review by awsdunc — APPROVED** [review]
LGTM

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 5]
```diff
@@ -279,6 +279,31 @@ This error is caused by there being too many [data source connectors](https://do
...
+### Option 1
+ To make this column show up in the CUR spin up a database in the RDS service, let it run for a couple of minutes and in the next integration of the crawler the column will appear. If you get the error that the column `product_cache_engine` does not exist, then you do not have any ElastiCache cache instances running. To make this column show up in the CUR spin up an ElastiCache cache instance in the ElastiCache service, let it run for a couple of minutes and in the next integration of the crawler the column will appear. You can verify this by running the Athena query: SHOW COLUMNS FROM tablename - and replace the tablename accordingly after selecting the correct CUR database in the dropdown on the left side in the Athena view.
+
+### Option 2
+Follow the below steps to remove the colum. Be aware this will mean if you do add ElastiCache instances to your accounts you should put this back.
+
+1. In Amazon Athena click 'Show Edit Query' for ``kpi_instance_all``
+2. Remove *`product_cache_engine`* and remove the last *Group by* number
+3. Run query
+4. IF you are running from CLoudShell re-run the deploy command
```
Comment (awsdunc) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:300]: Got some odd capitalisation happening in this line? 😆

Comment (awssteph) [reply content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md]: Done

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 6]
```diff
@@ -279,6 +279,31 @@ This error is caused by there being too many [data source connectors](https://do
...
+Follow the below steps to remove the colum. Be aware this will mean if you do add ElastiCache instances to your accounts you should put this back.
+
+1. In Amazon Athena click 'Show Edit Query' for ``kpi_instance_all``
+2. Remove *`product_cache_engine`* and remove the last *Group by* number
+3. Run query
+4. If you are running from CloudShell re-run the deploy command
+5. Go to Amazon Quicksight 
+6. Find the ``kpi_instance_all`` dataset
+7. Click on *'Edit Dataset'* 
+8. on the top right of the screen click *'save and publish'*
```
Comment (awsdunc) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:304]: On?

**Pull request merged**